### ISSUE TITLE
Secondary buttons, and primary button (disabled state) not WCAG 2.1 AA for colour contrast 

### ISSUE THREAD

**Issue by AndyPorter** [LandRegistry/hmlr-design-system-legacy#49]
Secondary buttons in both Default and disabled states do not meet AA for colour contrast.

default button meets AA if large text on button
fails normal size text (picture below)

disabled button fails both

Tested using Wave in Chrome.

Primary button in disabled mode also has insufficient colour contrast - for both normal and large text, but the Wave plugin does not report this - i tested it manually.

<img width="1162" alt="Screenshot 2019-08-15 at 15 01 54" src="https://user-images.githubusercontent.com/4385428/63099993-0b653280-bf6e-11e9-8d43-0cce7a4d682b.png">

**Comment by andymantell** [comment]
As per https://www.w3.org/TR/WCAG21/#contrast-enhanced

> Text or images of text that are **part of an inactive user interface component**, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, **have no contrast requirement.**

**Comment by AndyPorter** [comment]
secondary button - default state doesn't meet AA colour contrast.
